src/components/PeopleTable.tsx

```diff
@@ -10,44 +68,108 @@ export const PeopleTable = () => {
           <th>
             <span className="is-flex is-flex-wrap-nowrap">
               Name
-              <a href="#/people?sort=name">
-                <span className="icon">
-                  <i className="fas fa-sort" />
-                </span>
-              </a>
+              {sortParam !== 'name' && (
+                <SearchLink params={{ sort: 'name' }}>
+                  <span className="icon">
+                    <i className="fas fa-sort" />
+                  </span>
+                </SearchLink>
+              )}
+              {sortParam === 'name' && orderParam !== 'desc' && (
+                <SearchLink params={{ sort: 'name', order: 'desc' }}>
+                  <span className="icon">
+                    <i className="fas fa-sort-up" />
+                  </span>
+                </SearchLink>
+              )}
+              {sortParam === 'name' && orderParam === 'desc' && (
+                <SearchLink params={{ sort: null, order: null }}>
+                  <span className="icon">
+                    <i className="fas fa-sort-down" />
+                  </span>
+                </SearchLink>
+              )}
```

Comment:
try to simplify it by adding some conditions to the field that you changed
```suggestion
                <SearchLink params={getParams('name')}>
                  <span className="icon">
                    <i className={cn("fas", {...add some conditions here})} />
                  </span>
                </SearchLink>
```